### PR TITLE
feat(ios): add animation to annotations

### PR DESCRIPTION
**Pull request by anil-shukla-axway**
https://jira.appcelerator.org/browse/MOD-2450

### CODE REVIEW

File: ios/Classes/TiMapView.m [hunk 1]
```diff
@@ -1192,6 +1201,7 @@ - (MKAnnotationView *)mapView:(MKMapView *)mapView viewForAnnotationProxy:(TiMap
     annView.clusteringIdentifier = [ann valueForUndefinedKey:@"clusterIdentifier"];
     annView.collisionMode = [TiUtils intValue:[ann valueForUndefinedKey:@"collisionMode"]];
     annView.displayPriority = [TiUtils floatValue:[ann valueForUndefinedKey:@"annotationDisplayPriority"] def:1000];
+    [self animateAnnotation:annView withCoordinate:locationManager.location.coordinate];
```
Comment (vijaysingh-axway) [inline ios/Classes/TiMapView.m:1204]: Why animation is applied for iOS 11+ . It should be for iOS < 11.

**Comment by vijaysingh-axway** [comment]
@anil-shukla-axway  Should this PR be closed as I can see another PR #293 for same.

**Comment by anil-shukla-axway** [comment]
> @anil-shukla-axway Should this PR be closed as I can see another PR #293 for same.

Yes, this can be closed as I have created the other PR which covers both the cases for MOD-2418/2450.

**Comment by vijaysingh-axway** [comment]
Closing in favor of PR #293

**Pull request closed without merging**